src/java/components/hu.bme.mit.inf.modes3.components.dashboard/src/main/webapp/public_html/js/setup.js

```diff
@@ -4,6 +4,8 @@
  * and open the template in the editor.
  */
 
+var stream_url = "https://www.youtube.com/embed/5o_uF1L5l6o";
```

Comment:
This can be removed, because we won't stream on youtube

Reply:
How will the streaming be implemented then?

Reply:
The streaming will be implemented by WebRTC. Currently, I have a working example with one camera, and I will try out multiple camera streams on one page.
Problem is, I have implemented it in ES8/Babel, so integrating it will take a little time.

Reply:
Ok, then I remove every streaming-related implementation and leave that tab empty.